src/Sentry/Protocol/Replay.cs

```diff
@@ -0,0 +1,88 @@
+using Sentry.Extensibility;
+using Sentry.Internal;
+using Sentry.Internal.Extensions;
+
+namespace Sentry.Protocol;
+
+/// <summary>
+/// Sentry Replay context interface.
+/// </summary>
+/// <example>
```

Comment:
nitpick: Would you mind wrapping the code in the example with `<code> .. </code>`?